examples/plugin_new_macro.py

```diff
@@ -1,7 +1,7 @@
 # Compatible with ranger 1.6.0 through 1.7.*
 #
 # This plugin adds the new macro %date which is substituted with the current
-# date in commands that allow macros.  You can test it with the command.
+# date in commands that allow macros.  You can test it with the command:
```

Comment:
This still reads weird. Either rephrase or don't change it.

Reply:
I haven't changed it with the exception of the colon at the end. I just added a few lines  below to clarify how macros can be functions or values. Should I remove those?

Reply:
No, this one sentence is the problem. "The command" and the colon don't mix well. Either drop the colon or do something like "You can test it with: "